src/defaultsettings.cpp

```diff
@@ -440,8 +440,8 @@ void set_default_settings()
 	settings->setDefault("chat_message_limit_per_10sec", "5.0");
 	settings->setDefault("chat_message_limit_trigger_kick", "50");
 	settings->setDefault("sqlite_synchronous", "2");
-	settings->setDefault("map_compression_level_disk", "-1");
-	settings->setDefault("map_compression_level_net", "-1");
+	settings->setDefault("map_compression_level_disk", "0");
+	settings->setDefault("map_compression_level_net", "0");
```

Comment:
It turns out this isn't necessary, the compression level is increased by 1 when using zstd so that -1 is still the default level:

https://github.com/MultiCraft/MultiCraft/blob/ac4e39d0b94d278a72bee6fb9cda3d952c5028cc/src/serialization.cpp#L306-L315